config/default.yaml

```diff
@@ -100,3 +100,6 @@ logging:
   rotate:
     size: 52428800 # bytes, default is 50mb
     count: 5
+
+whiteboard:
+  whiteboardUrl: "https://whiteboard-dev.nordeck.net"
```

Comment:
This should be the default instance of the project https://cloud13.de/testwhiteboard

Reply:
actually i think that we don't have to push any code to default.yaml at least first for the pull request to dimesnion. it's a file that we can edit on the server i quess. or on our build(the ame for dev.yaml annd stage.yaml)

Reply:
what you mean by default has to be in the optionsJson